shared/common-adapters/animated.js.flow

```diff
@@ -0,0 +1,8 @@
+// @flow
+import * as React from 'react'
+export type Props = {|
+  children?: Function,
```

Comment:
should be something tighter like
```
children: (styles: CrossPlatformStyles) => React.Node
```

Reply:
It's different between desktop and native.